mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1227,32 +1248,23 @@ def _get_args_for_metrics(self, extra_metric, eval_df):
                     # case where the param is defined as part of the evaluator_config
                     elif column in self.evaluator_config:
                         eval_fn_args.append(self.evaluator_config.get(column))
+
+                    # case where this is the name of another metric
+                    elif column in self.metrics_values:
+                        eval_fn_args.append(self.metrics_values[column])
+
+                    # only for checking dependencies
+                    elif column in [metric_tuple.name for metric_tuple in self.ordered_metrics]:
+                        eval_fn_args.append(None)
```

Comment:
What's going on here? `append(None)` seems a bit confusing. Can we make the comment clearer?